Readme.md

```diff
@@ -1,19 +1,59 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128570167/21.1.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T381476)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+
+# How to Show a Custom Context/Popup Menu for Certain Chart Elements
+
+This example demonstrates how to use [Menus](https://docs.devexpress.com/WPF/115388/controls-and-libraries/ribbon-bars-and-menu/menus) and [Bars](https://docs.devexpress.com/WPF/6194/controls-and-libraries/ribbon-bars-and-menu/bars) to show a custom context menu for chart series. 
+
+![Resulting chart](images/resulting-chart.png)
+
+Use the [BarManager.DXContextMenu](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.BarManager.DXContextMenu) attached property to attach a context menu to a chart control. 
```

Comment:
`[BarManager.DXContextMenu] attached property` Is this called an attached property? Or is 'attached' the type of property, in which case it should be 'attached [name of property] property'

Reply:
Microsoft uses "the [Property Name] attached property" wording.
For example:
https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.automationproperties.positioninset?view=windowsdesktop-6.0
An [attached property](https://docs.microsoft.com/fi-fi/dotnet/desktop/wpf/advanced/attached-properties-overview?view=netframeworkdesktop-4.8) is a term. It seems to me that "attached property" should go together after the property name.

Reply:
Ok